This ensures that libmutter_cogl_path_dep is built before other targets,
fixing the build with many buildjobs

Obtained from:	Aports

--- clutter/clutter/meson.build.orig	2020-09-07 19:18:23 UTC
+++ clutter/clutter/meson.build
@@ -424,7 +424,7 @@ libmutter_clutter = shared_library(libmutter_clutter_n
   soversion: 0,
   c_args: clutter_c_args,
   include_directories: clutter_includes,
-  dependencies: [clutter_deps],
+  dependencies: [clutter_deps,libmutter_cogl_path_dep],
   gnu_symbol_visibility: 'hidden',
   link_with: [
     libmutter_cogl,
@@ -436,7 +436,7 @@ libmutter_clutter = shared_library(libmutter_clutter_n
   install: true,
 )
 libmutter_clutter_dep = declare_dependency(
-  sources: [clutter_enum_types[1]],
+  sources: [clutter_enum_types[1],libmutter_cogl_path_enum_types[1]],
   link_with: libmutter_clutter,
   dependencies: clutter_deps,
 )
